lib/frame.rb

```diff
@@ -0,0 +1,16 @@
+class Frame
+
+    attr_reader :total, :roll_1, :roll_2
+  
+    def initialize(roll_1:, roll_2: 0)
+      @roll_1 = roll_1
+      @roll_2 = roll_2
+      @total = calculate()
+      return @total
+    end
+  
+    def calculate
```

Comment:
you have another calculate method in another class. Not sure what is best practice here but could this get confusing?